### PR TITLE
msize opcode modification

### PR DESCRIPTION
**Pull request by ashWhiteHat**
Part of https://github.com/privacy-scaling-explorations/zkevm-specs/issues/230

I introduced `StepState` and `Instruction` to `MSIZE` opcode.
The circuit constraints and logics are aligned with zkevm-circuits.
https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/execution/msize.rs#L30

I renamed StateStep `memory_size` to `memory_word_size` to align with zkevm-circuits.
https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/step.rs#L452

I would appreciate it if you could confirm.
Thank you!

### CODE REVIEW

File: specs/opcode/16AND_17OR_18XOR.md [hunk 1]
```diff
@@ -36,4 +36,4 @@ apply the lookup to the 32 chunks of `a`, `b`, and `c` to see if
 
 ## Code
 
-See `src/zkevm_specs/opcode/bitwise.py`
+See `src/zkevm_specs/evm/execution/comparator.py`
```
Comment (han0110) [inline specs/opcode/16AND_17OR_18XOR.md:39]:
```suggestion
See `src/zkevm_specs/evm/execution/bitwise.py`
```

**Review by han0110 — APPROVED** [review]
LGTM! Nice refactor!

**Comment by ashWhiteHat** [comment]
Hi @han0110 
Thank you for the review.
I fixed!

**Review by ed255 — APPROVED** [review]
LGTM! Thanks for the clean up :)

**Pull request merged**